### PR TITLE
[SC-111] refactor to remove nested stacks

### PR DESCRIPTION
**Pull request by zaro0508**
Nested stacks make things confusing and makes cloudformation
very difficult to test.  We remove nested stack and will deploy
stacks with sceptre.  Scepter will manage the dependencies between
stacks.

### CODE REVIEW

File: ec2/sc-product-ec2-linux-jumpcloud-notebook.yaml [hunk 1]
```diff
@@ -4,12 +4,7 @@ Parameters:
   PortfolioProvider:
     Type: String
     Description: Owner and Distributor Name
-  LaunchConstraintARN:
-    Type: String
-    Description: ARN of the launch constraint role for EC2 products.
-  PortfolioId:
-    Type: String
-    Description: The ServiceCatalog portfolio this product will be attached to.
+    Default: IT Services
```
Comment (ahayden) [inline ec2/sc-product-ec2-linux-jumpcloud-notebook.yaml:7]: Maybe call this `IT Operations`?

File: ec2/sc-product-ec2-windows-jumpcloud.yaml [hunk 2]
```diff
@@ -43,16 +38,18 @@ Resources:
...
     Properties:
-      PortfolioId: !Ref 'PortfolioId'
+      PortfolioId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-portfolio-ec2-SCEC2portfolioId'
       ProductId: !Ref 'scec2windowsproduct'
-      RoleArn: !Ref 'LaunchConstraintARN'
-      Description: !Ref 'LaunchConstraintARN'
+      RoleArn: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-ec2vpc-launchrole-LaunchRoleArn'
 Outputs:
```
Comment (ahayden) [inline ec2/sc-product-ec2-windows-jumpcloud.yaml:53]: I think you need to export the `ProductId`.

File: ec2/sc-product-ec2-linux-jumpcloud-notebook.yaml (hunk 1 above)
Comment (zaro0508) [reply ec2/sc-product-ec2-linux-jumpcloud-notebook.yaml:7]: good idea, will do on a follow on PR.

File: ec2/sc-product-ec2-windows-jumpcloud.yaml (hunk 2 above)
Comment (zaro0508) [reply ec2/sc-product-ec2-windows-jumpcloud.yaml:53]: good catch.  i've been neglecting windows.

**Pull request merged**